### PR TITLE
ADEN-11599 fix nativo events handling

### PR DESCRIPTION
**Pull request by jakubbanas**

### CODE REVIEW

File: src/ad-services/nativo/index.ts [hunk 1]
```diff
@@ -23,12 +29,17 @@ export class Nativo {
...
 			.then(() => {
 				utils.logger(logGroup, 'ready');
+				this.watchNtvEvents();
 				this.sendNativoLoadStatus(AdSlot.SLOT_ADDED_EVENT);
 			});
 	}
 
-	private handleNativoNativeEvent(e, slot: AdSlot, adStatus: string) {
-		utils.logger(logGroup, 'Nativo native event fired', e, adStatus);
+	private handleNtvNativeEvent(e, slotName: string, adStatus: string) {
```
Comment (preludek) [inline src/ad-services/nativo/index.ts:37]: That method has a weird name imo. `handleNativoResponseEvent` maybe? Or `handleNativoAdEvent`

File: src/ad-services/nativo/index.ts [hunk 2]
```diff
@@ -1,16 +1,22 @@
 import { communicationService, eventsRepository } from '@ad-engine/communication';
 import { AdSlot, context, slotService, utils } from '@ad-engine/core';
-import { logger } from '../../ad-engine/utils';
```
Comment (preludek) [inline src/ad-services/nativo/index.ts:3]: 💎

File: src/ad-services/nativo/index.ts [hunk 3]
```diff
@@ -1,16 +1,22 @@
...
 export const libraryUrl = 'https://s.ntv.io/serve/load.js';
 
 export class Nativo {
 	static INCONTENT_AD_SLOT_NAME = 'ntv_ad';
 	static FEED_AD_SLOT_NAME = 'ntv_feed_ad';
+
 	static SLOT_CLASS_LIST = ['ntv-ad', 'ad-slot'];
 	static TEST_QUERY_STRING = 'native_ads_test';
 
+	private static AD_SLOT_MAP = {
```
[the remark this answers is not in the document]
Comment (nandy-andy) [reply src/ad-services/nativo/index.ts:14]: Correct! We can but I'd leave it within this file, so all stuff connected to Nativo is in one place ;) We'll discuss it more while talking about my "big refactoring" 😊 And we'll change it once we'll add more incontents (I recall Lisandro has it planned for the future) or in the "big refactoring" if we agree to have stuff in different places ;)

File: src/ad-services/nativo/index.ts (hunk 1 above)
Comment (nandy-andy) [reply src/ad-services/nativo/index.ts:37]: Why is it weird? It's Nativo's native events :p I wanted to make it consistent with newly added `watchNtvEvents()` ;)

File: src/ad-services/nativo/index.ts (hunk 2 above)
Comment (nandy-andy) [reply src/ad-services/nativo/index.ts:3]: THIS

File: src/ad-services/nativo/index.ts (hunk 1 above)
Comment (preludek) [reply src/ad-services/nativo/index.ts:37]: Tongue twister :D

Comment (nandy-andy) [reply src/ad-services/nativo/index.ts:37]: Tru dat. I'll re-think it during the refactoring ;)

**Pull request merged**